src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/Events.table.tsx

```diff
@@ -0,0 +1,64 @@
+import moment from 'moment'
```

Comment:
Why are we keeping the name `.table`  in the component name. Can we rename it to EventTable.tsx or something like that